idd/Energy+.idd.in

```diff
@@ -38873,34 +38870,22 @@ AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan,
        \minimum 0.0
        \maximum 1.0
        \note fraction of cooling air flow rate
-  A3,  \field Air Inlet Node Name
```

Comment:
@Nigusse After some discussion with @kbenne we've decided to keep the Air Inlet Node Name and Air Outlet Node Name fields for AirTerminal:SingleDuct:VAV:Reheat:VariableSpeedFan to maintain consistency across terminal units.  Please remove the \deprecated tag from these and undo the idf file and doc changes just for these two fields - sorry for the change in plan.  Now the interesting part is that getinput is not currently reading these two fields, it mines them from the fan and reheat coil.  Options:
1.  Leave getinput as-is and have fields that could hold garbage and throw no errors.
2.  Add a check to see if the input field matches the equivalent field that is mined from the child objects and throw a warning if they don't match.  If we go that route, then we need a transition rule that finds the child objects and makes sure these fields are filled with the correct node names.
@Myoldmopar Any preference?